content/applications/inventory_and_mrp/inventory/management/products/uom.rst

```diff
@@ -2,106 +2,111 @@
+be sold, which is also the unit in which internal transfers will take place. You can also select
+the *Purchase Unit of Measure*, which is the unit in which you purchase the product.
 
-.. image:: uom/uom_04.png
-    :align: center
+.. image:: uom/internal-uom-vs-purchase-uom.png
+   :align: center
+   :alt: Specify Units of Measure by use: internal transfer or when the product is purchased.
 
 Transfer from One Unit to Another
 =================================
 
 Buy in the Purchase UoM
 -----------------------
 
-In the *Purchase* application, *Create* a new request for quotation
-in which you include the product with the different *Units of Measure*
-and *Confirm* it.
+In the *Purchase* application, *Create* a new request for quotation in which you include the
+product with the different *Units of Measure* and *Confirm* it.
 
-.. image:: uom/uom_05.png
-    :align: center
+.. image:: uom/rfq-with-diff-uom.png
+   :align: center
+   :alt: A request for quotation illustrating different Units of Measure.
 
-On the automatically generated purchase orders, the UoM used is the Box
-of 6, meaning the Purchase UoM. You have of course the possibility to
-manually modify the UoM if necessary. When you enter the *Receipt*
-which is linked to the purchase order, you can observe that the 10 boxes
-of 6 units have been converted in 60 units. Indeed, the stock is managed
-in units.
+On the automatically generated purchase orders, the UoM used is the Box of 6, meaning the Purchase
+UoM. You have of course the possibility to manually modify the UoM if necessary. When you enter the
+*Receipt* which is linked to the purchase order, you can observe that the 10 boxes of 6 units have
+been converted in 60 units. Indeed, the stock is managed in units.
 
-.. image:: uom/uom_06.png
-    :align: center
+.. image:: uom/uom-conversion-on-receipt.png
+   :align: center
+   :alt: Units are converted on purchase order receipts.
 
 Replenishment
 -------------
 
-When doing a replenishment via the *Replenish* button on the product
-form, you have the possibility to use a different unit of measure.
+When doing a replenishment via the *Replenish* button on the product form, you have the possibility
+to use a different unit of measure.
 
-.. image:: uom/uom_07.png
-    :align: center
+.. image:: uom/replenish-button-on-product.png
+   :align: center
+   :alt: The Replenish button on the product allows for different units of measure.
 
-.. image:: uom/uom_08.png
-    :align: center
+.. image:: uom/replenish-product-uom-settings.png
+   :align: center
+   :alt: Product replenishment UoM settings.
 
 Sell in bigger UoM
 ------------------
 
-You can choose the unit of measure on the sale order document and decide
-to sell the eggs by the dozen. When doing so, the price is automatically
-computed from Units UoM to adapt to the selected *UoM*.
+You can choose the unit of measure on the sale order document and decide to sell the eggs by the
```

Comment:
- Delete the two images `sales-order-uom.png` (Line 103) and `delivery-order-uom-conversion.png` (Line 110)
- Replace the three sentences on Lines 99-101 and Lines 107-108 `You can choose...in our use case, the Units.` with:

`When creating a new quotation in the Sales app, Odoo automatically uses the product's specified unit of measure. However, if needed, the :guilabel:'UoM' can be manually edited on the quotation.` 
[line break]
`After the quotation is sent to the customer and confirmed into a sales order (SO), click on the :guilabel:'Delivery' smart button at the top right corner of the SO. Odoo automatically converts the unit of measure into the product's inventory unit of measure, so the :guilabel:'Demand' column of the delivery shows the converted quantity.`
[line break]
`For example, if the product's UoM on the SO was changed to 'Box of 6', but its inventory unit of measure is 'Units', the SO shows the quantity in boxes of six, and the delivery shows the quantity in units.`  

(^ single quotation marks = backticks in my suggested change)